src/index.js

```diff
@@ -36,6 +36,6 @@ export {
 	crypto,
 	transaction,
 	api,
-	slots,
+	time,
```

Comment:
Do we definitely want these exported here?

Reply:
Yes, I think it is helpful. Why should we not export it? Or do you mean belonging to the transaction export?